coralproject/talk
# [CORL-2555]: fix adding an expert in Q&A

## What does this PR do?

These changes make it so that it's possible to add an expert in Q&A mode again. With changes to shadow dom, `window.document` needed to either be updated to `root`, or we use the `ClickOutside` component to control clicking outside of the expert selection search. These changes update to use the `ClickOutside` component.

## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags? 

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

You can go to `Configure` for a story. Switch to Q&A mode. Search for an expert. Click on the expert you want to add and see that they appear as added under `Experts`. Also clicking outside of the search list when it's open should close it.
 
## How do we deploy this PR?
